must-gather/collection-scripts/gather_log

```diff
@@ -43,8 +43,8 @@ done
 echo "INFO: Gathering logs of ODF Block Driver"
 ODF_PODS=($(oc -n ${ODF_NAMESPACE} get pods --no-headers -o custom-columns=":metadata.name" -l odf="storage.ibm.com"))
 for ODF_POD in ${ODF_PODS[@]}; do
```

Comment:
I do not have much idea about must-gather, But I still think that this should be fully automated and should not ask for any input from users.

Reply:
Yes, gather logs actions don't ask user for any input.

Reply:
As far as I understand this we are totally omitting the name of the container from this. So the body of commit-msg is not leading in the right direction. IMO we should correct the body as it won't be defined by the user.
![Screenshot from 2021-07-12 14-31-24](https://user-images.githubusercontent.com/22993933/125260481-fd68e980-e31d-11eb-9a2b-5a20bb48ca96.png)